### PR TITLE
[Feat] Add "lamas" logo to anyway card

### PR DESCRIPTION
**Pull request by ialish**

### CODE REVIEW

File: src/components/atoms/LamasLogo.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import LamasImage from '../../assets/cbs.png';
+
+interface LamasLogoProps {
+  height: string;
+}
+
+export const LamasLogo: FC<LamasLogoProps> = ({ height }) => {
+  const logo = useMemo(() => LamasImage.toString(), []);
+
+  const logoStyle = useMemo(
```
Comment (yuvalbl) [inline src/components/atoms/LamasLogo.tsx:11]: Why [memoize](https://reactjs.org/docs/hooks-reference.html#usememo) a `prop`?
Make sure you understand the hook before using it :)

File: src/components/atoms/LamasLogo.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React, { FC, useMemo } from 'react';
+import LamasImage from '../../assets/cbs.png';
+
+interface LamasLogoProps {
+  height: string;
+}
+
+export const LamasLogo: FC<LamasLogoProps> = ({ height }) => {
+  const logo = useMemo(() => LamasImage.toString(), []);
```
Comment (yuvalbl) [inline src/components/atoms/LamasLogo.tsx:9]: Why `useMemo`??

File: src/components/molecules/AnyWayCard.tsx [hunk 3]
```diff
@@ -48,7 +49,12 @@ const AnyWayCard: FC<IProps> = ({children}) => {
         <AnyWayButton className={classes.button } onClick={imgDownloadHandler}>
           <GetAppOutlinedIcon />
         </AnyWayButton>
-        <AnywayLogo height={'25px'} />
+				<div>
+					<div style={{ display: 'inline', margin: '10px' }}>
```
Comment (yuvalbl) [inline src/components/molecules/AnyWayCard.tsx:53]: never use inline styles.
see [useStyles](https://material-ui.com/styles/basics/) hook

File: src/components/atoms/LamasLogo.tsx (hunk 1 above)
Comment (ialish) [reply src/components/atoms/LamasLogo.tsx:11]: I've duplicate the AnywayLogo component and adjust it to the LamasLogo.

**Pull request merged**